src/main.tsx

```diff
@@ -1,24 +1,31 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import App from './App.tsx'
-import {
-  createBrowserRouter,
-  RouterProvider,
-} from "react-router-dom";
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import App from "./App.tsx";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import ErrorPage from "./error-page";
-import './index.css'
+import "./index.css";
+import { ThemeProvider } from "styled-components";
 
 const router = createBrowserRouter([
   {
     path: "/",
     element: <App />,
-    errorElement: <ErrorPage />
+    errorElement: <ErrorPage />,
   },
 ]);
 
+const theme = {
+  /* The following are Spotify's primary branding colors as shown in their 
+  design guidelines https://developer.spotify.com/documentation/design#using-our-colors */
+  green: "#1ED760",
+  white: "#FFFFFF",
+  black: "#121212",
+};
 
-createRoot(document.getElementById('root')!).render(
+createRoot(document.getElementById("root")!).render(
   <StrictMode>
-    <RouterProvider router={router} />
-  </StrictMode>,
-)
+    <ThemeProvider theme={theme}>
```

Comment:
This `ThemeProvider` gives us access to the theme in all styled components we define in the app